app/templates/src/styles/main.styl

```diff
@@ -1,63 +1,90 @@
 // New to Stylus? Check out http://learnboost.github.io/stylus
 // Use modern CSS thanks to Autoprefixer: https://github.com/ai/autoprefixer
 
-<% if (syntax || !useTheme) { -%>
+<% if (syntax) { -%>
 // Import CSS from "node_modules"
 // thanks to Stylus' "import css" and "paths" options
 <% } -%>
-<% if (!useTheme) { -%>
 @import 'normalizecss/normalize.css'
-<% } -%>
 <% if (syntax) { -%>
 // Theme with really good contrast FTW!
 @import 'prism-themes/themes/prism-ghcolors.css'
 // Check out "node_modules/prismjs/themes/" and "node_modules/prism-themes/themes/" for available themes
 <% } -%>
-<% if (!useTheme) { -%>
 // Style using Bespoke Classes: https://github.com/markdalgleish/bespoke-classes
 
+*
+::before
+::after
+  box-sizing inherit
```

Comment:
I prefer one line per selector. I find it easier to remove/reorder/comment one selector (in general) and easier to read.

I changed it to comma-separated.